ui/components/app/security-provider-banner-alert/security-provider-banner-alert.js

```diff
@@ -48,31 +50,53 @@ function SecurityProviderBannerAlert({
       )}
 
       {provider && (
-        <Text
-          marginTop={3}
-          display={Display.Flex}
-          alignItems={AlignItems.center}
-          color={Color.textAlternative}
-          variant={TextVariant.bodySm}
-        >
-          <Icon
-            className="disclosure__summary--icon"
-            color={IconColor.primaryDefault}
-            name={IconName.SecurityTick}
-            size={IconSize.Sm}
-            marginInlineEnd={1}
-          />
-          {t('securityProviderPoweredBy', [
-            <ButtonLink
-              key={`security-provider-button-link-${provider}`}
-              size={Size.inherit}
-              href={SECURITY_PROVIDER_CONFIG[provider].url}
-              externalLink
-            >
-              {t(SECURITY_PROVIDER_CONFIG[provider].tKeyName)}
-            </ButtonLink>,
-          ])}
-        </Text>
+        <>
+          <Text
+            marginTop={3}
+            display={Display.Flex}
+            alignItems={AlignItems.center}
+            color={Color.textAlternative}
+            variant={TextVariant.bodySm}
```

Comment:
```suggestion
```

In the design, it looks like it should match the text above. If so, I think we can remove these. I think `alignItems` was only needed below to align the icon in the center of the lines